model/publishing/delivery/tasks/DeployTestEnvironments.php

```diff
@@ -101,7 +101,8 @@ protected function compileTest(core_kernel_classes_Resource $env, core_kernel_cl
                     'name' => RestTest::REST_DELIVERY_PARAMS,
                     'contents' => json_encode(
                         [
-                            PublishingDeliveryService::ORIGIN_DELIVERY_ID_FIELD => $delivery->getUri()
+                            PublishingDeliveryService::ORIGIN_DELIVERY_ID_FIELD => $delivery->getUri(),
+                            PublishingDeliveryService::ORIGIN_TEST_ID_FIELD => $test->getUri()
```

Comment:
Using this inserts PublishingDeliveryService::ORIGIN_TEST_ID_FIELD property on statements for remote delivery and not the test. To accommodate it on the test we need to make minor modifications on [ImportAndCompile](https://github.com/oat-sa/extension-tao-delivery-rdf/blob/master/model/tasks/ImportAndCompile.php#L72) task